src/PantherTestCaseTrait.php

```diff
@@ -162,21 +162,21 @@ public static function isWebServerStarted()
      */
     protected static function createPantherClient(array $options = [], array $kernelOptions = []): PantherClient
     {
-        $browser = ($options['browser'] ?? self::$defaultOptions['browser'] ?? self::CHROME);
+        $browser = ($options['browser'] ?? self::$defaultOptions['browser'] ?? PantherTestCase::CHROME);
         $callGetClient = \is_callable([self::class, 'getClient']) && (new \ReflectionMethod(self::class, 'getClient'))->isStatic();
         if (null !== self::$pantherClient) {
             $browserManager = self::$pantherClient->getBrowserManager();
             if (
-                (self::CHROME === $browser && $browserManager instanceof ChromeManager) ||
-                (self::FIREFOX === $browser && $browserManager instanceof FirefoxManager)
+                (PantherTestCase::CHROME === $browser && $browserManager instanceof ChromeManager) ||
+                (PantherTestCase::FIREFOX === $browser && $browserManager instanceof FirefoxManager)
```

Comment:
We could also move the constants to this trait. Since this is the only thing that uses these constants, it would make sense to define them here as well.